lib/ansible/module_utils/ibm_sa_utils.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2018 IBM CORPORATION
+# Author(s): Tzur Eliyahu <tzure@il.ibm.com>
+#
```

Comment:
As `module_utils/*` is BSD licensed, need to replace line 3 to 15 to 

```
# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
```

Reply:
Hi, I'm not a legal expert but I can see that there are others under the module_utils folder which use the gnu license (like docker common, cloud, ovirt, vca and etc...), so why is this a problem? changing it now requires my legal team approval, which will take time...

Reply:
Done